doc/api/index.rst

```diff
@@ -24,6 +24,7 @@ Forward modelling
 
     point_mass_gravity
     tesseroid_gravity
+    distance
```

Comment:
I think this probably shouldn't be in the public AP if it's a utility meant for use in the forward modeling modules. We can always make it public later but removing the function is harder.